backend/src/main/java/de/neuefische/backend/security/SecurityConfig.java

```diff
@@ -42,6 +43,8 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                             ("/user/login"), ("/user/register")).permitAll();
                     auth.requestMatchers(
                             ("/user/**")).authenticated();
+                    auth.requestMatchers(
+                            ("/locations/**")).authenticated();
                     auth.anyRequest().permitAll();
```

Comment:
anyRequest should never be permitAll. Let be denyAll and you know when didn't apply authentication properly